x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -17,7 +17,7 @@ export const allowedExperimentalValues = Object.freeze({
   tGridEventRenderedViewEnabled: true,
   excludePoliciesInFilterEnabled: false,
   usersEnabled: true,
-  detectionResponseEnabled: false,
+  detectionResponseEnabled: true,
```

Comment:
We better wait until is done to enable this one :)
I always enable the flags using:
```
xpack.securitySolution.enableExperimental: ['detectionResponseEnabled']
```
in my `kibana.dev.yml`. 
Instead of changing this file, so I do not enable flags accidentally.

Reply:
Yeah, thanks, enabled it for deploying to cloud instance, will remember to disable that

Reply:
Oh, that makes completelly sense, sorry I overthought it 👍